src/commands/exile_commands.py

```diff
@@ -31,7 +31,7 @@ async def unexile(interaction: discord.Interaction, user: discord.Member):
     @discord.app_commands.check(is_user_moderator)
     @discord.app_commands.describe(
         user="User being exiled",
-        duration="The duration of the exile. TBA format",
+        duration="The duration of the exile. Duration should be in the form of [1 or 2 digits][s, d, m, h]",
```

Comment:
Unless you are very technically inclined, this is awful to read as an end user. My suggestion is: 
`Enter a value Between 1 and 99 followed by s, m, h, or d. Ex: 1s, 1m, 1h, 1d.`

Also, i think we should think about changing the the characters after the numerical value. M can be mistaken for mins or months..

I feel they should be:
- sec
- min
- hour
- day

This would then change the tooltip to:
`Enter a value Between 1 and 99 followed by sec, min, hour, or day. Examples: 1sec, 1min, 1hour, 1day.`

Reply:
@clairefleuret I think this makes a lot of sense, but I'm gonna pull it into its own PR since it's changing actual code rather than just a comment

Reply:
If you can make a ticket that would be great.